api.code-workspace

```diff
@@ -3,23 +3,5 @@
 		{
 			"path": "."
 		}
-	],
-	"settings": {
-		"editor.codeActionsOnSave": {
-			"source.fixAll.eslint": true
-		},
-		"[javascript]": {
-			"editor.tabSize": 2,
-			"editor.insertSpaces": true,
-			"editor.detectIndentation": false,
-			"editor.formatOnPaste": true,
-			"editor.formatOnSave": true
-		},
-		"markdownlint.config": {
-			"default": true,
-			"MD003": { "style": "setext_with_atx" },
-			"MD034": false,
-			"MD046": false
-		}
```

Comment:
why did you remove this?

Reply:
it is added back into a separate settings.json file so you can open one workspace with all the projects.

Reply:
ok, I see. Upon merging this, we will need to tell the rest of the team, because otherwise this may cause confusion -- just post on slack #general telling people that this has been changed and how to use the settings.json file